tasks/seed.js

```diff
@@ -21,13 +22,13 @@ export async function runSetup() {
  meetingIds: [ObjectId]
 */
   let dt = new Date();
-
+  const hashPW = await bcrypt.hash("abcDefgh2i",16);
   const sampleUser = {
     first_name: "Sample",
     last_name: "User",
     email: "sampleUser@gmail.com",
     // password will be hashed when being passed from UI to the API and then stored ( done by Jonathan)
-    password: "sampleUser",
+    password: hashPW,
     disability: false,
     // date string passed here is MM/DD/YYYY
     dob: new Date("01/01/1996"),
```

Comment:
doesnt this need to be a string ? 
lets discuss

Reply:
i like bread